Modules/LearningSequence/classes/class.ilObjLearningSequenceAdminGUI.php

```diff
@@ -0,0 +1,143 @@
+<?php declare(strict_types=1);
+
+use ILIAS\UI\Component\Input;
+
+/**
+ * LearningSequence Administration Settings
+ *
+ * @ilCtrl_Calls ilObjLearningSequenceAdminGUI: ilPermissionGUI
+ */
+class ilObjLearningSequenceAdminGUI extends ilObjectGUI
+{
+    const CMD_VIEW = 'view';
+    const CMD_EDIT = 'edit';
+    const CMD_SAVE = 'save';
+    const F_POLL_INTERVAL = 'polling';
+
+    /**
+     * @var ilLSGlobalSettingsDB
+     */
+    protected $settings_db;
+
+    public function __construct($a_data, $a_id, $a_call_by_reference = true, $a_prepare_output = true)
```

Comment:
Da wir den constructor überschreiben sollten wir hier auch typehinten können. Ausserdem können wir in einem Zug auch die $a_ Prefixe los werden.